tests/data/CN.mm9.toy_regions.bed

```diff
@@ -1,2 +1,2 @@
 chr1	100000000	150000000	foo
-chr2	100000000	150000000	bar
+chr2	100000000	150000000	bar
```

Comment:
does black do this ?
i'm not an expert on all things unix, but i think there should be a new line at the end e.g. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see.

Reply:
Maybe I did myself by accident...